oci/oci-explanation/ubuntu-oci-container.rst

```diff
@@ -0,0 +1,22 @@
+.. _ubuntu-oci-container-images:
+
+Ubuntu OCI container images
+===========================
+
+The Open Container Initiative (OCI) establishes standards for constructing container 
+images that can be reliably installed across a variety of compliant host environments.
+
+Ubuntu’s `LTS Docker Image Portfolio <https://ubuntu.com/security/docker-images>`_ 
+provides OCI-compliant images that receive stable security updates and predictable 
+software updates, thus ensuring consistency in both maintenance schedule and operational 
+interfaces for the underlying software your software builds on.
```

Comment:
The double use of "software" reads a bit strangely. I'd probably change the first one to something else.